client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js

```diff
@@ -612,27 +617,25 @@ function CloudLink(props) {
     response
   } = props;
 
-  const {
-    camundaCloudClusterRegion,
-    camundaCloudClusterId
-  } = endpoint;
-
   const processId = getProcessId(response);
 
   if (!processId) {
     return null;
   }
 
-  const query = `?process=${processId}&version=all&active=true&incidents=true`;
-  const cloudUrl = `https://${camundaCloudClusterRegion}.operate.camunda.io/${camundaCloudClusterId}/instances/${query}`;
+  const cloudUrl = getCloudLink(endpoint, response);
+  cloudUrl.searchParams.set('process', processId);
+  cloudUrl.searchParams.set('version', 'all');
+  cloudUrl.searchParams.set('active', 'true');
+  cloudUrl.searchParams.set('incidents', 'true');
```

Comment:
Nice stuff!

Reply:
Yeah I think it's safer than manual typing the queries.